src/cursor.c

```diff
@@ -362,45 +361,56 @@ cursor_update_common(struct server *server, struct cursor_context *ctx,
 		return;
 	}
 
-	if (ctx->surface && !input_inhibit_blocks_surface(seat,
-			ctx->surface->resource)) {
+	if (!ctx->surface) {
+		/*
+		 * Cursor is over a server (labwc) surface.  Clear focus
```

Comment:
Suggest changing `server (labwc) surface` to something more descriptive. I'm not sure what.
Is it server-side-deco only?

Reply:
Could also be the root/desktop, menus, etc.

Reply:
Well maybe not menus -- I guess those are an interactive mode and handler earlier.

Reply:
"server (labwc) owned surface"

Reply:
Reading it through I just wasn't clear on what it means, so I thought that's a good reason for being quite explicit in the comment - maybe be giving an example. Having said that... I don't feel strongly - so go with what you think.

It's slightly non-intuitive that when `ctx->surface==NULL` it might still be a surface just one that is owned by the server.
I haven't looked, but guess this is relating to the clash of surface/node/SDD-enums in the ...node_at() function?

Reply:
Agree.. that is indeed a bit strange. Not sure what to do to make it more clear though.